<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>13</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8bit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>load 8_bit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12bit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>540</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>load 12_bit</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlide_start">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>540</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1024</number>
   </property>
   <property name="maximum">
    <number>3071</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_width">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>570</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4095</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_start">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>540</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_width">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>570</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3d">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>540</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Load3D</string>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider_schichten">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="maximum">
    <number>129</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_schicht">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>420</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bild:</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_schwellenwert">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>600</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1024</number>
   </property>
   <property name="maximum">
    <number>3071</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_schwellenwert">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>590</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>schwellenwert:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tiefenkarte">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tiefenkarte</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
